### PR TITLE
UI - Manage stake modal components 

### PR DESCRIPTION
**Pull request by masvelio**
## Link to Issue
Closes: #5476 #5558 

## Description of Changes
- added flow and components necessary for hooking up buying and selling stake
- added new loading spinner component
- logic should be added in #6330 

https://github.com/hicommonwealth/commonwealth/assets/14819225/ee5b22e2-5150-4428-8536-21d33a87a7dc

## Test Plan
- testing should happen when everything will be hooked up, for now it is only a static, mocked data

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/TransactionSucceeded/TransactionSucceeded.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            {app?.chain?.meta?.name}
+          </CWText>
+          {conditionalText.descriptionSuffix}
+        </CWText>
+
+        <CWDivider className="divider" />
+
+        <div className="under-divider">
+          <CWText type="caption">
+            Don’t trust, verify. Track your transaction on Etherscan.
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/TransactionSucceeded/TransactionSucceeded.tsx:78]: `&apos;` for `'`

File: packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/TransactionSucceeded/TransactionSucceeded.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  };
+
+  const handleCopyLink = async () => {
+    try {
+      await navigator.clipboard.writeText('link to copy');
+      notifySuccess('Link copied');
+    } catch (err) {
+      console.log(err);
+    }
+  };
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/TransactionSucceeded/TransactionSucceeded.tsx:38]: we make this into a `clipboard.ts` util

**Review by mzparacha — APPROVED** [review]
Looks great, Thanks!

**Review by lzach83 — APPROVED** [review]
Fantastic job

File: packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/TransactionSucceeded/TransactionSucceeded.tsx (hunk 2 above)
Comment (masvelio) [reply packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/TransactionSucceeded/TransactionSucceeded.tsx:33]: done

**Pull request merged**